packages/components/src/alignment-matrix-control/utils.js

```diff
@@ -0,0 +1,55 @@
+export const GRID = [
+	[ 'top left', 'top center', 'top right' ],
+	[ 'center left', 'center center', 'center right' ],
+	[ 'bottom left', 'bottom center', 'bottom right' ],
+];
+
+// Transforms GRID into a flat Array of values
+export const ALIGNMENTS = GRID.reduce( ( alignments, row ) => {
+	row.forEach( ( cell ) => {
+		alignments.push( cell );
+	} );
+
+	return alignments;
+}, [] );
+
+/**
+ * Parses and transforms an incoming value to better match the alignment values
+ *
+ * @param {string} value An alignment value to parse.
+ *
+ * @return {string} The parsed value.
+ */
+export function parseValue( value ) {
```

Comment:
Maybe it should have a better name to communicate that it's replacing `-` by spaces?